spec/dummy/db/migrate/3_create_payload_hypertable.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
 class CreatePayloadHypertable < ActiveRecord::Migration[Rails.version[0..2]]
-  def up
-    execute 'CREATE SEQUENCE payload_id_seq'
+  disable_ddl_transaction!
```

Comment:
Why do we need this one here?

Reply:
forgot to remove it after doing some tests with indexes